src/config/clusters/clusters.config.ts

```diff
@@ -4,10 +4,10 @@ import GRPC_SERVICES_NAMES from '../grpc/grpc-services-names';
 import CLUSTER_NAMES from './cluster-names.config';
 import { type ClusterConfig, type ClustersConfigs } from './clusters.types';
 
-const configsHasSameLength = [GRPC_PEERS, GRPC_SERVICES_NAMES].every(
+/*const configsHasSameLength = [GRPC_PEERS, GRPC_SERVICES_NAMES].every(
```

Comment:
What do we want to do about this eventually?

Reply:
This is a composite issue:
- first we need to split clusters config to UI & backend configurations to prevent leaking clusters config to UI
- second is check only on the backend the configuration issue.